docs/tutorials/contracts/fts/simple-fts.md

```diff
@@ -0,0 +1,155 @@
+  "E-mail Account Recovery", although "Recovery Phrase" and [Ledger](https://www.ledger.com/)
+  are the most secure methods.
+
+#### 3. E-mail / Phone Number Account Recovery
+
+- Enter the account activation code that you received.
+
+#### 4. Success!
+
+- You just created a `testnet` account and received 200 Ⓝ! Upon recovery method confirmation
+  you should be directed to your account dashboard.
+
+Once you have your Wallet account, you can click on the [Balances Tab](https://wallet.testnet.near.org/?tab=balances) where all your Fungible Tokens will be listed:
+
+![Wallet](/docs/assets/fts/empty-wallet-ft-tab.png)
+
+### Installing the `near-cli`
+
+The following instructions are taken from the `near-cli` [installation
+guide](https://docs.near.org/docs/tools/near-cli#setup). If you already have the command line
+interface, you can [skip these steps](#using-the-ft-contract).
+
+> **Note:** Make sure you have a current version of `npm` and `NodeJS` installed.
+
+#### Linux and macOS
+
+1. Install `npm` and `node` using a package manager such as `nvm`. Sometimes there are issues
+   using Ledger due to how macOS handles node packages related to USB devices.
+   [[click here]](https://nodejs.org/en/download/package-manager/)
+2. Ensure you have installed Node version 12 or above.
+3. Install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
+
+#### Windows
+
+> **Note:** For Windows users, we recommend using Windows Subsystem for Linux (`WSL`).
+
+1. Install `WSL` [[click here]](https://docs.microsoft.com/en-us/windows/wsl/install-manual#downloading-distros)
+2. Install ` Node.js` [[click here]](https://nodejs.org/en/download/package-manager/)
+3. Change `npm` default directory [[click here]](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory)
+   - This is to avoid any permission issues with `WSL`
+4. Open `WSL` and install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
+
+## Using the FT contract
+
+Now that you have all the tools in place, you're ready to start using the FT smart contract to [mint your FTs](#minting-your-fts).
+
+Log in to your newly created account with `near-cli` by running the following command in your terminal.
+
+```bash
+near login
+```
+
+To make this tutorial easier to copy/paste, we're going to set an environment variable for your account ID. In the command below, replace `YOUR_ACCOUNT_NAME` with the account name you just logged in with including the `.testnet` (or `.near` for `mainnet`):
+
+```bash
+export ID=YOUR_ACCOUNT_NAME
+```
+
+Test that the environment variable is set correctly by running:
+
+```bash
+echo $ID
+```
+
+### Minting your FTs
+
+NEAR has already deployed a contract to the account `ft.examples.testnet` which allows users to freely mint `TeamTokens`. This is the account we'll be interacting with to mint our FTs.
```

Comment:
```suggestion
NEAR has already deployed a contract to the account `ft.examples.testnet` which allows anyone to freely mint `TeamTokens`. This is the account you'll be interacting with to mint your fungible tokens.
```

Best practice is to refer to the user by using "your" and "you'll" instead of "we" and "we'll"